skflow/__init__.py

```diff
@@ -51,11 +51,12 @@ class TensorFlowEstimator(BaseEstimator):
 
     def __init__(self, model_fn, n_classes, tf_master="", batch_size=32, steps=50, optimizer="SGD",
                  learning_rate=0.1, tf_random_seed=42, continue_training=False,
-                 log_device_placement=True):
+                 log_device_placement=True, verbose=1):
```

Comment:
Yeah, we may also want to let users choose how many steps before it prints out some results, e.g. `if step % print_steps == 0` or `if step % print_steps == 10`.